mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -2,7 +2,6 @@
 
 from mapie._machine_precision import EPSILON
 from mapie._typing import ArrayLike, NDArray
-
 from mapie.conformity_scores import ConformityScore
```

Comment:
GLOBAL COMMENT: try to separate type and global imports from mapie imports and other library imports, each separated by a new line.

Reply:
Prefer the previous version.